pkg/converter/decoder.go

```diff
@@ -57,15 +57,18 @@ func FromJSON(b []byte) *Decoder {
 // extension.
 func FromFileName(fname string, contents []byte) *Decoder {
 	ext := filepath.Ext(fname)
+	var d *Decoder
```

Comment:
I know you're a heavy vimmer, how do you deal with these short variable names I'm seeing used so often around here?   
breaks all the ways I know how to use my tools

Reply:
ha. I don't use refactoring tools very often. if I'm using a single-variable name, it's because it's very locally scoped. Which tools does it break, out of curiosity?